### PR TITLE
add storage stats card to My LBRY downloads page

### PR DESCRIPTION
**Pull request by akinwale**

### CODE REVIEW

File: app/src/component/storageStatsCard/view.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+import { normalizeURI, parseURI } from 'lbry-redux';
+import {
+  ActivityIndicator,
+  Platform,
+  Switch,
+  Text,
+  TouchableOpacity,
+  View
+} from 'react-native';
+import { formatBytes } from '../../utils/helper';
```
Comment (skhameneh) [inline app/src/component/storageStatsCard/view.js]: We really need to get aliases or resolves added to remove these.
I die a little inside every time I see these crazy paths, assign an issue to me if you need me to work on this.

File: app/src/utils/helper.js [hunk 2]
```diff
@@ -1,40 +1,59 @@
 import { NavigationActions, StackActions } from 'react-navigation';
 
-export function navigateToUri(navigation, uri, additionalParams) {
-    if (!navigation) {
+export function dispatchNavigateToUri(dispatch, nav, uri) {
+  const params = { uri };
+  if (nav && nav.routes && nav.routes.length > 0 && 'Main' === nav.routes[0].routeName) {
```
Comment (skhameneh) [inline app/src/utils/helper.js]: Can you add more helper methods for detecting the state of routes?
This is a lot of fragile code that can quickly become unmanageable.

File: app/src/utils/helper.js [hunk 3]
```diff
@@ -1,40 +1,59 @@
...
-        navigation.dispatch(stackAction);
-        return;
-    }
+export function formatBytes(bytes, decimalPoints = 0) {
+  if (!bytes) {
+    return '0 KB';
+  }
 
-    navigation.navigate({ routeName: 'File', key: uri, params });
+  if (bytes < 1048576) { // < 1MB
```
Comment (skhameneh) [inline app/src/utils/helper.js]: Can define constants for a lot of these that are reused.

File: app/src/component/storageStatsCard/view.js (hunk 1 above)
Comment (akinwale) [reply app/src/component/storageStatsCard/view.js]: I plan to sort this out later. I just haven't taken the time to figure out how to do this yet.

**Pull request merged**